src/__tests__/old-act.js

```diff
@@ -98,4 +98,22 @@ test('async act recovers from sync errors', async () => {
   `)
 })
 
+test('async act handles values that are not strings', async () => {
+  try {
+    await asyncAct(() => {
+      throw new Error({error: 'test error'})
```

Comment:
I think you have to do a `console.error({ message: 'some message' }`  insteadof throwing it, to mimic the problem

Reply:
Changed the test to not throw error.

Reply:
This is correct 👍 

but you had to change the assertion of another test, which is an indication of a wrong implementation (you are still not calling the original console.error)

check this implementation for inpiration :-):
https://github.com/rvdkooy/react-testing-library/commit/9020b83ca047c6dadaaa1e70407b6146caf5cff9

@kentcdodds can you have a look if we're on the right track with this?